.github/workflows/e2e_tests.yml

```diff
@@ -0,0 +1,105 @@
+name: E2E Tests
+
+on:
+  pull_request:
+
+concurrency:
+  group: ${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+  
+jobs:
+  run-e2e:
+    name: Install Datalens and Run E2E Tests
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          repository: datalens-tech/datalens
+          path: datalens
+      - uses: actions/checkout@v4
+        with:
+          path: datalens-ui
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 18
+          cache: 'npm'
+          cache-dependency-path: ./datalens-ui/package-lock.json
+
+      - name: Run Docker Compose
+        working-directory: ./datalens
+        run: docker compose -f ./docker-compose.yml -f ../datalens-ui/docker-compose.yml up -d & cd ../datalens-ui && npm ci && npm run test:install:chromium & wait
+
+
+      # - name: Install dependencies Datalens-UI 
+      #   working-directory: ./datalens-ui
+      #   run: npm ci && npm run test:install:chromium
+      - name: Run Test 
+        working-directory: ./datalens-ui
+        run: E2E_DOMAIN="http://localhost:8080" E2E_TEST_NAME_PATTERN="Open page" npm run test:e2e
```

Comment:
Now we start just one test. We need to start all *opensource* tests. We can:

1. Put them to separate folder and grep by folder name
2. Use tags https://playwright.dev/docs/test-annotations#tag-tests
3. Name opensource tests with different prefix and grep tests by test name.